lib/defaults.js

```diff
@@ -0,0 +1,10 @@
+'use strict';
+
+module.exports = {
+  CANT_FIND: 'I can\'t find the address in the list',
+  CHANGE: 'Change',
+  POSTCODE_ERROR: {
+    'not-found': 'Sorry – we couldn’t find any addresses for that postcode, enter your address manually',
+    'cant-connect': 'Sorry – we couldn’t connect to the postcode lookup service at this time, enter your address manually'
+  }
+}
```

Comment:
if lookup fails (you forget or CBA to include a translation value) these are used as defaults. I think a standalone component/behaviour should have fallbacks for things like these to minimise setup in the majority of use cases